subsys/nrf_security/src/zephyr/CMakeLists.txt

```diff
@@ -34,10 +34,12 @@ if(CONFIG_MBEDTLS_ENTROPY_POLL)
   )
 endif()
 
-# Include a late initialization of psa_crypto_init just-in-case
-list(APPEND src_zephyr
-  psa_crypto_init.c
-)
+if(CONFIG_PSA_CRYPTO_SYS_INIT)
+  # Include a late initialization of psa_crypto_init just-in-case
+  list(APPEND src_zephyr
+	psa_crypto_init.c
```

Comment:
either put all on one line or indent this line

Reply:
fixed